### ISSUE TITLE
Make Station name on station overview page a clickable link

### ISSUE THREAD

**Issue by j-sayers** [MotusDev/Motus-TO-DO#1264]
The only way currently to access the direct link for a station is to view the deployment details, where the name of the station is a clickable link.

But for stations without deployments this isn't available. Plus it would just be far easier to have this at the level of the station itself rather than having to click down first.

In the [example ](https://motus.org/data/station/deployment/edit?projectID=183&id=7573)below the station is linked in the deployment details.

![image](https://user-images.githubusercontent.com/55001144/193867769-9204b107-2c74-4274-b1ca-f35b8c92da83.png)

But the station itself should also have the direct link. 

![image](https://user-images.githubusercontent.com/55001144/193868263-fc351a37-b7a0-436a-825c-8b9ced20f8b1.png)

**Comment by bcreightonyoung** [comment]
@j-sayers, could you clarify where you would like this clickable link to redirect the user to? I tried to follow your example, but clicking the station link as in the first image brings you to the second image. So I can't quite figure out where clicking the station name in the second image is meant to send the user.

**Comment by j-sayers** [comment]
Yeah a bit confusing... It's basically a link to the same page (the second image). The quickest way to get the shareable link now is to click one of the deployments, and from there get the link. But if the link were on the station overview page too it would save that extra step.

**Comment by bcreightonyoung** [comment]
@j-sayers I think I maybe understand what you are saying... On sandbox I added a link pointing to itself on the station summary pages. Once I realized that your goal is to get a url pointing to a specific station the issue made more sense. Let me know if I'm still missing something!
